openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.mode_changed.json

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Modified",
+  "actor": {
+    "id": "http://localhost:18000/u/honor",
+    "name": "honor",
+    "type": "Person"
+  },
+  "eventTime": "2018-11-16T07:52:15.893Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9",
+      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
+      "course_id": "course-v1:PUCIT+CS520+2018_T4",
```

Comment:
I believe `course_id` and `org_id` should be moved inside `extensions` key of `object`

Reply:
Done